src/service/ampdoc-impl.js

```diff
@@ -425,13 +425,13 @@ export class AmpDoc {
    * @return {?string}
    */
   getMetaByName(name) {
-    if (!name) {
+    if (!name || !this.win.document || !this.win.document.head) {
```

Comment:
Can `this.win.document` be null?

Reply:
How could `head` be nonexistent? I thought even an html document without a `<head>` tag will still be given an empty one.

Reply:
In practice, this is only for unit tests. See for example https://github.com/ampproject/amphtml/blob/3d4ed6244a94951d85bcf5bd384e83916fc56e6f/test/unit/test-url-replacements.js#L156 , where the `document` object is defined by hand. I'll defer to you two on whether this change should be merged; I'm impartial.